arm_parameters: 
  elbow:
    encoder_parameters:
      counts_per_rotation: 1000
      offset: 500
    motor_parameters:
      gear_ratio: 3
  forearmRoll:
    encoder_parameters:
      counts_per_rotation: 1000
      offset: 500
    motor_parameters:
      gear_ratio: 3
  shoulder:
    encoder_parameters:
      counts_per_rotation: 1000
      offset: 500
    motor_parameters:
      gear_ratio: 9
  turntable:
    encoder_parameters:
      counts_per_rotation: 1000
      offset: 500
    motor_parameters:
      gear_ratio: 3
  wristPitch:
    encoder_parameters:
      counts_per_rotation: 1000
      offset: 500
    motor_parameters:
      gear_ratio: 2
  wristRoll:
    encoder_parameters:
      counts_per_rotation: 1000
      offset: 500
    motor_parameters:
      gear_ratio: 2
